test/http_connection_info_test.dart

```diff
@@ -39,15 +38,13 @@ Future<Null> testHttpConnectionInfo() {
       response.listen((_) {}, onDone: () {
         client.close();
         server.close();
-        completer.complete(null);
+        completer.complete();
       });
     });
   });
   return completer.future;
 }
 
 void main() {
-  test("HttpConnectionInfo", () async {
-    await testHttpConnectionInfo();
-  });
+  test('HttpConnectionInfo', () => testHttpConnectionInfo());
```

Comment:
This could be just test('HttpConnectionInfo', testHttpConnectionInfo);

Reply:
That's actually a static error in strong mode.

```error • The argument type 'Future<Null>' can't be assigned to the parameter type '() → dynamic' at test/http_connection_info_test.dart:49:30 • argument_type_not_assignable```